### DISCUSSION TITLE
Manual/job-system-jobs

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#526] [Notes]
# Manual/job-system-jobs

https://docs.unity3d.com/Manual/job-system-jobs.html

**Comment by radiatoryang** [comment]
This blog post "Job Types in the Unity Job System" by Sebastian Schöner gives much more detail on the various job types and how to use them: http://blog.s-schoener.com/2019-04-26-unity-job-zoo/
